extensions/wikia/Email/Controller/EmailConfirmationController.class.php

```diff
@@ -36,15 +36,16 @@ public function assertCanEmail() {
 	}
 
 	/**
-	 * @template emailConfirmation
+	 * @template otherEmailConfirmation
```

Comment:
👍 for combing these into one class. Think we could still update the name of the template? It looks like this template is used for the emailConfirmation email the user receives right after registration? Maybe we could rename it to `registrationEmailConfirmation` or `initialEmailConfirmation`?